### PR TITLE
AutoPong sample added

### PR DESCRIPTION
**Pull request by MrGrak**
Hi! I'd like to add a sample game to this repo. It's a simple example of pong that plays itself called AutoPong. I think it offers a very simple introduction to game design and managing game state, perfect for beginners to take apart and play with. It also shows a very simple way to get soundfx playing in a game. The game has some unique properties, for example no content is loaded from hard drive. Instead, the graphics, soundfx and musical jingle is created using only code. It's also relatively small, at around 330 lines total. This should work cross platform, assuming that SpriteBatch + DynamicSoundEffectInstance have been implemented on that platform. Cheers

### CODE REVIEW

**Review by damian-666 — COMMENTED** [review]
i like this. I didnt know there was a Synthesizer.. should break that into a separate file though.

File: AutoPong/Game1.cs [hunk 1]
```diff
@@ -0,0 +1,328 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Audio;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+
+namespace Game1
```
Comment (SimonDarksideJ) [inline AutoPong/Game1.cs:7]: Needs to have a unique namespace specific to the sample

**Review by SimonDarksideJ — CHANGES_REQUESTED** [review]
Thanks for submitting this, great to have another / simpler sample to show learners.
But please make sure it's a complete sample, including a solution and building for as many platforms as possible.

Also, if you can add some additional docs walking through the sample, explaining why things are done and how it works, so it becomes a fantastic learning resource.

Great work.
(P.S. apologies for only just seeing this)

**Comment by SimonDarksideJ** [comment]
Thanks for the sample @MrGrak , it has been incorporated in to the 3.8.1 sample set, which is now live

**Pull request closed without merging**